src/transformers/modeling_auto.py

```diff
@@ -166,6 +167,10 @@
     XLNetLMHeadModel,
     XLNetModel,
 )
+from .modeling_prophetnet import (
+    ProphetNetModel,
+    ProphetNetForConditionalGeneration
+)
```

Comment:
I totally understand that `ProphetNet` is a specialized model for generation but you may also want to add something like `ProphetNetForSequenceClassification` etc.? `BART` has them.

Reply:
Thank you for this suggestion. I will add ProphetNetForSequenceClassification in near future when I find a suitable way for NLU tasks. For example, I will compare different encoder-decoder NLU functions of T5, BART...

Reply:
Agree that it would be nice to also have `ProphetNetForSequenceClassification`, but I think we could handle this in a new PR. The most important is probably `ProphetNetForConditionalGeneration` for now